core/modules/contact/contact.pages.inc

```diff
@@ -76,6 +76,13 @@ function contact_site_form($form, &$form_state) {
     '#default_value' => $user->uid ? $user->mail : '',
     '#required' => TRUE,
   );
+  if (!empty($config->get('phone'))) {
```

Comment:
These are the killers. See https://joomla.stackexchange.com/questions/5565/fatal-error-cant-use-method-return-value-in-write-context

Reply:
Thanks @indigoxela & @docwilmot!